docs/website/content/v0.3/en/guides/cloud/aws.md

```diff
@@ -2,18 +2,225 @@
 title: 'AWS'
 ---
 
-First, create the AMI:
+## Creating a Cluster via the AWS CLI
+
+In this guide we will create an HA Kubernetes cluster with 3 worker nodes.
+We assume an existing VPC.
+
+### Create the Subnet
+
+```bash
+aws ec2 create-subnet \
+    --region ${REGION} \
+    --vpc-id ${VPC} \
+    --cidr-block 10.0.1.0/24
+```
+
+### Create the AMI
+
+#### Prepare the Import Prerequisites
+
+##### Create the S3 Bucket
+
+```bash
+aws s3api create-bucket \
+    --bucket talos-aws-tutorial-${ACCOUNT} \
+    --create-bucket-configuration LocationConstraint=${REGION} \
+    --acl private
+```
+
+##### Create the `vmimport` Role
+
+In order to create an AMI, ensure that the `vmimport` role exists as described in the [official AWS documentation](https://docs.aws.amazon.com/vm-import/latest/userguide/vmie_prereqs.html#vmimport-role).
+
+##### Create the Image Snapshot
+
+```bash
+curl -LO https://github.com/talos-systems/talos/releases/download/v0.3.0-alpha.2/aws.tar.gz
+tar -xvf aws.tar.gz
+aws s3 cp aws.raw s3://${BUCKET}/talos-aws-tutorial.raw
+aws ec2 import-snapshot \
+    --region ${REGION} \
+    --description "Talos kubernetes tutorial" \
+    --disk-container "Format=raw,UserBucket={S3Bucket=${BUCKET},S3Key=talos-aws-tutorial.raw}"
```

Comment:
looks like `$BUCKET` should be  `talos-aws-tutorial-${ACCOUNT}` from the above?

Reply:
Good catch. I think I want to keep it generic so I will stick with `$BUCKET`